pubspec.yaml

```diff
@@ -2,21 +2,27 @@ name: flutter_gutter
 description: Get your UI out of the gutter! Ensure all visual gaps between your widgets are consistent, adapted to the
   axis direction, and respond to screen size.
 repository: https://github.com/caseycrogers/flutter_gutter
+issue_tracker: https://github.com/caseycrogers/flutter_gutter/issues
+homepage: https://github.com/caseycrogers/flutter_gutter
+topics:
+  - gap
+  - gutter
 
-version: 1.0.3
+version: 1.1.0
```

Comment:
With the removal of block this is a technically a breaking change, and with the really expansive additions to the API I think a major version bump is pretty justified here, lets make this `2.0.0`?

Reply:
What do you mean with the removal of block? I tried to keep the breaking changes as minimal as possible, at least to the api surface. But there are breaking changes for sure so a 2.0.0 version is good too.

Let's make sure to include all the other changes you wanted to do first before release.

Reply:
sorry "removal of Gap" I'm extra tired today.

Reply:
Aah right. In that case, I thought about removing Margin as well since it has no added value. What do you think about that?

Reply:
I lean towards leaving Margin in, especially because, while it's always just identical to a Gutter today, I've been meaning to release an update that allows you to manually write your own breakpoint -> pixel size mappings via the `GutterConfiguration` system at which point your Gutter and Margin could be different sizes.